fe/fe-core/src/main/java/org/apache/doris/nereids/properties/SelectHintSetVar.java

```diff
@@ -50,6 +50,9 @@ public Map<String, Optional<String>> getParameters() {
      * @param context statement context
      */
     public void setVarOnceInSql(StatementContext context) {
+        if (context == null) {
```

Comment:
I think the root cause maybe is the code in org.apache.doris.mtmv.MTMVCache#from

the code as following
```
        StatementContext mvSqlStatementContext = new StatementContext(connectContext,
                new OriginStatement(mtmv.getQuerySql(), 0));
        NereidsPlanner planner = new NereidsPlanner(mvSqlStatementContext);
        if (mvSqlStatementContext.getConnectContext().getStatementContext() == null) {
            mvSqlStatementContext.getConnectContext().setStatementContext(mvSqlStatementContext);
        }
```

should before` LogicalPlan unboundMvPlan = new NereidsParser().parseSingle(mtmv.getQuerySql())`;

then the StatementContext is not null in current connect Context

Reply:
thanks, I have modified it, please help review it.